python/core/advanced-queues/prioritize-your-queue.md

```diff
@@ -70,6 +70,36 @@ class Enki(object):
 
 To give an example of when it is advised to use `PriorityQueue`, we can thing of a hospital. Here sick people come with no interruption. Now, in a ideal situation, the hospital's staff would prioritize people depending on how sever their illness is. If there is someone who suffers a headache should be treated only after one whose heart has stopped working was treated. In such a case we would use a `PriorityQueue` treating instantly all sever illnesses and put the others in the queue.
 
+---
+## Practice
+
+Fill the following code snippet such that it will sort elements added in the queue in descending order:
+
+```python
+
+def Order():
```

Comment:
Also create a code example to verify that this is how it works, with http://pythontutor.com/ here and link to the example in the links section.

Reply:
Unfortunately I can't use http://pythontutor.com/  here, because they don't offer support for the queue module. Basically I can't import it here, however I checked it on the REPL on my laptop and works fine. There was a type tho, it's not `Order` is not a method, but a class. Thanks for the review!